### PR TITLE
Add SRG and Mojmap aliases for OptionListWidgetMixin8.optionsScreen

### PR DESCRIPTION
**Pull request by unilock**
Allows running the mod properly on NeoForge (via [Sinytra Connector](https://github.com/Sinytra/Connector)), which uses some weird amalgamation of Mojmap and SRG during runtime (to my recollection). More specifically, it fixes a crash on 1.21(.1) when opening the "Controls" options menu.

### CODE REVIEW

**Comment by murmuur22** [comment]
How can I download the jar for your fix?

**Comment by murmuur22** [comment]
Also thanks btw ur actualy saving me i keep accidently crashing on my friends server 🙏

File: src/main/java/com/hamarb123/macos_input_fixes/mixin/gui/OptionListWidgetMixin8.java [hunk 1]
```diff
@@ -13,7 +13,8 @@
 @Mixin(OptionListWidget.class)
 public class OptionListWidgetMixin8
 {
-	@Shadow(remap = false, aliases = {"field_49483", "optionsScreen"})
+	// Intermediary, Yarn, SRG, Mojmap
```
Comment (hamarb123) [inline src/main/java/com/hamarb123/macos_input_fixes/mixin/gui/OptionListWidgetMixin8.java:16]:
```suggestion
	//Intermediary, Yarn, SRG, Mojmap
```

**Comment by hamarb123** [comment]
Sorry for the delay in getting to this - I was busy at the time, and then forgot to look at it :)
Is this the only place something like this is needed? If so, why (so I can hopefully pre-emptively avoid the issue in the future)?
Thanks!

**Comment by unilock** [comment]
> Is this the only place something like this is needed? If so, why...?

Well, this is the only usage of `remap = false` that causes a crash on NeoForge 1.21.1 specifically. I suppose that every other use of aliases (incl. `method = {...}` with multiple values) could include SRG and Mojmap names, too, for the sake of future-proofing.

**Comment by hamarb123** [comment]
It seems to be the only `remap = false` that gets used in 1.21+, other than ScreenMixin.java and HandledScreenMixin.java (interesting that these didn't cause an issue too). And what website/tool can I use to get the Mojmap/SRG mappings from intermediary name or similar, do you know?

**Comment by murmuur22** [comment]
thanks so much y'all saved me, ur mod is a requirement for me. :)

**Comment by unilock** [comment]
Yes, I fully agree to abide by the Contributor License Agreement ("CLA") as listed at the provided Internet link https://gist.github.com/hamarb123/a1cf2a35cb9dfb75dd219fbfc655817c.

**Pull request merged**